error-handling/index.js

```diff
@@ -0,0 +1,21 @@
+const { isDarkmode } = require('../middleware/middleware.js');
+
+module.exports = (app) => {
+  app.use(isDarkmode, (req, res, next) => {
```

Comment:
why is this darkmode stuff in errorhandling?

Reply:
If you are in Dark mode, I guess you want dark error messages...